.github/workflows/test.yml

```
@@ -0,0 +1,44 @@
+name: "Test all features"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@b173b6ec0100793626c2d9e6b90435061f4fc3e5 # v0.11.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Install updates and protobuf-compiler
+        run: sudo apt update && sudo apt install --assume-yes cmake protobuf-compiler
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: wasm32-unknown-unknown
+
+      - name: Install nightly toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+          target: wasm32-unknown-unknown
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Fetch cache
+        uses: Swatinem/rust-cache@6fd3edff6979b79f87531400ad694fb7f2c84b1f # v2.2.1
+
+      - name: Test all features
+        run: cargo test --workspace --all-features
```

Okay the `all-features` was maybe not as smart as first thought. Can we try the *exact* same as in Polkadot? 
  Will also need `RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"`. 
```suggestion
        run: cargo test --workspace --profile testnet --locked --features=runtime-benchmarks,runtime-metrics,try-runtime
```
 For whatever reason it compiles fine with `try-runtime` enabled but not with `--all-features` :see_no_evil:

---

This worked locally for me.

---

Done, applied.